index.js

```diff
@@ -31,7 +36,11 @@ function dnxRunner(dnxCommand, options) {
   }
 
   if (options.run === true) {
-    commands.push('@powershell -NoProfile -ExecutionPolicy unrestricted -Command "for(;;) { Write-Output \"Starting...\"; dnx --watch . ' + dnxCommand + ' }"');
+    if (isWin) {
+      commands.push(util.format('@powershell -NoProfile -ExecutionPolicy unrestricted -Command "for(;;) { Write-Output \"Starting...\"; dnx --watch . %s }"', dnxCommand));      
+    } else {
+      commands.push(util.format('nodemon --ext "cs,json" --exec "dnx . %s"', dnxCommand));
```

Comment:
Nice to use nodemon!

This requires nodemon globally though. Can we use the one which is under our dependency graph?